reference/services/git-hosting.rst

```diff
@@ -0,0 +1,92 @@
+Git hosting
+===============
+
+Description
+-----------
+
+Launchpad has direct Git hosting support alongside its existing Bazaar hosting support. 
+The major subsystems involved are:
+
+* The git client
+* Hosting service (turnip)
+* The underlying object model
+* The web application (Launchpad)
+* Repository source code browser (cgit)
+
+Git client
+----------
+
+This is what users install on their systems to use Git. 
+It is also installed on the hosting backend to support the ``receive-pack`` and ``upload-pack`` protocols.
+
+Hosting service
+---------------
+
+The outside world connects to the Git hosting service using one of several protocols:
+
+* ``git://``
+* ``git+ssh://``
+* ``https://``
+
+The Git "dumb" HTTP transport will not be supported, as it has largely been considered 
+deprecated since the provision of "smart" HTTP transport in Git v1.6.6 (released January, 2010).
+
+SSL is terminated by haproxy (haproxy has two units haproxy/1* and haproxy/0) for HTTPS, and all the other 
+protocols also pass through haproxy for load balancing (although at the moment we only have a single backend).  
+See the full charm configuration at `launchpad-mojo-specs <https://code.launchpad.net/launchpad-mojo-specs>_`.
```

Comment:
Very minor, just noticed that the _ at the end of the link should be outside of the ` :)
Also, perhaps we could link to the bundle directly maybe?
https://git.launchpad.net/launchpad-mojo-specs/tree/lp-git/bundle.yaml